web/src/typing.js

```diff
@@ -6,14 +6,24 @@ import * as blueslip from "./blueslip";
 import * as channel from "./channel";
 import * as compose_pm_pill from "./compose_pm_pill";
 import * as compose_state from "./compose_state";
+import {page_params} from "./page_params";
 import * as people from "./people";
 import {user_settings} from "./user_settings";
 
 // This module handles the outbound side of typing indicators.
 // We detect changes in the compose box and notify the server
 // when we are typing.  For the inbound side see typing_events.js.
 //
+
 // See docs/subsystems/typing-indicators.md for details on typing indicators.
+// How frequently 'still typing' notifications are sent
+// to extend the expiry
+export const TYPING_STARTED_WAIT_PERIOD =
```

Comment:
Why are these exported?

Also, I don't think they should be capitalized since they are coming from the server.